pandas/core/generic.py

```diff
@@ -8875,13 +8875,20 @@ def _add_series_only_operations(cls):
         def nanptp(values, axis=0, skipna=True):
             nmax = nanops.nanmax(values, axis, skipna)
             nmin = nanops.nanmin(values, axis, skipna)
+            warnings.warn("Method .ptp is deprecated and will be removed "
+                          "in a future version. Use numpy.ptp instead.",
+                          FutureWarning, stacklevel=4)
             return nmax - nmin
 
         cls.ptp = _make_stat_function(
             cls, 'ptp', name, name2, axis_descr,
-            """Returns the difference between the maximum value and the
+            """
+            .. deprecated:: 0.24.0
+                Use numpy.ptp instead
+            Returns the difference between the maximum value and the
```

Comment:
This needs a blank line above this one. Can you also put the deprecated part below the text?

Reply:
Thanks - Done - let me know if the deprecated note below main text needs any edits, have also added the line `Use numpy.ptp instead` again